client/data/segmentaton-survey/queries/use-survey-structure-query.ts

```diff
@@ -36,8 +37,14 @@ const mapSurveyStructureResponse = ( response: SurveyStructureResponse ): Questi
 	} );
 
 const useSurveyStructureQuery = ( { surveyKey }: SurveyStructureQueryArgs ) => {
+	// This follows exact logic as addLocaleQueryParam() middleware
+	// which will add _locale to the query param and we want to cache
+	// our surveys with by the same locale value.
```

Comment:
Small comment copy suggestion:

```suggestion
	// our surveys with the same locale value.
```